tc/sgx/trusted_worker_manager/enclave/enclave_data.cpp

```diff
@@ -57,6 +57,9 @@ EnclaveData::EnclaveData(void) {
     // Create the public encryption key
     public_encryption_key_ = private_encryption_key_.GetPublicKey();
 
+    /* Create encryption key signature */
```

Comment:
For consistency, use the same comment style as the rest of the file.  That is:
`//`

Reply:
Ok